app/models/invenio_rdm_record_converter.rb

```diff
@@ -246,14 +244,17 @@ def build_creator_contributor_json(creator)
           "given_name": given_name,
           "family_name": family_name,
           "type": "personal",
-          "identifiers": identifiers
         }
       }
+
+      if dh_user.orcid.present?
+        identifiers = [{"scheme": "orcid", "identifier": dh_user.orcid.split('/').pop}]
+        json.merge!({"identifiers": identifiers})
```

Comment:
`"identifiers": identifiers` should be merged in `"person_or_org"` value (not top-level).